pubspec.yaml

```diff
@@ -12,6 +12,6 @@ dependencies:
   yaml: ^3.1.0
 
 dev_dependencies:
-  ffigen: ^3.0.0
+  ffigen: ^6.1.2
```

Comment:
You should update the minimum SDK version (line 7) as well. The generated code requires Dart SDK 2.13.0 (or 2.17.0) or higher.